### ISSUE TITLE
Combine modes

### ISSUE THREAD

**Issue by skywluke** [Anonym-tsk/homeassistant-climate-xiaomi-remote#7]
Hi! I would like to ask you, is there any way to combine all 3 modes in one commmand? Because to my daikin not possible to send independent commands. So i think the following:
'off':
cool:
  low:
    comfort:
      18: 
      19:
    boost:
      18: 
      19:
Thank you very much for answer

**Comment by Anonym-tsk** [comment]
Which 3 modes do you want to combine? cool + low + 18 - it's possible

**Comment by swissjon** [comment]
Yes skywluke, that is my understanding for daikin and LG at least. When you send a command from an AC remove, it sends all modes in one blast. 
For example: 24C, fan low, current time, timer length, swing. All sent as one string/blast. 
24C, *fan high*, current time, timer length, swing would be a different string. It doesn't send multiple commands for each state/mode you want to update

So I wonder how this app could work with one code per mode/state. Unless I am getting this completely wrong. I have searched for documentation on how AC manufacturers create their IR strings but haven't found anything yet.
